dotCMS/src/main/java/com/dotmarketing/cms/urlmap/URLMapInfo.java

```diff
@@ -9,10 +9,12 @@
 public class URLMapInfo {
     final Contentlet contentlet;
     final Identifier identifier;
+    final String urlMapped;
 
-    URLMapInfo(final Contentlet contentlet, final Identifier identifier) {
+    URLMapInfo(final Contentlet contentlet, final Identifier identifier, String urlMapped) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'urlMapped' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4842460)